diff --git a/node_modules/@remix-run/react/dist/components.d.ts b/node_modules/@remix-run/react/dist/components.d.ts
index fd88339..61989f0 100644
--- a/node_modules/@remix-run/react/dist/components.d.ts
+++ b/node_modules/@remix-run/react/dist/components.d.ts
@@ -44,7 +44,7 @@ export declare function composeEventHandlers<EventType extends React.SyntheticEv
  *
  * @see https://remix.run/components/links
  */
-export declare function Links(): React.JSX.Element;
+export declare function Links({ nonce}: { nonce?: string }): React.JSX.Element;
 /**
  * This component renders all the `<link rel="prefetch">` and
  * `<link rel="modulepreload"/>` tags for all the assets (data, modules, css) of
diff --git a/node_modules/@remix-run/react/dist/components.js b/node_modules/@remix-run/react/dist/components.js
index d0ba14b..8ebe7bd 100644
--- a/node_modules/@remix-run/react/dist/components.js
+++ b/node_modules/@remix-run/react/dist/components.js
@@ -191,7 +191,7 @@ function composeEventHandlers(theirHandler, ourHandler) {
  *
  * @see https://remix.run/components/links
  */
-function Links() {
+function Links({ nonce} ) {
   let {
     manifest,
     routeModules,
@@ -204,6 +204,7 @@ function Links() {
   let matches = errors ? routerMatches.slice(0, routerMatches.findIndex(m => errors[m.route.id]) + 1) : routerMatches;
   let keyedLinks = React__namespace.useMemo(() => links.getKeyedLinksForMatches(matches, routeModules, manifest), [matches, routeModules, manifest]);
   return /*#__PURE__*/React__namespace.createElement(React__namespace.Fragment, null, criticalCss ? /*#__PURE__*/React__namespace.createElement("style", {
+    nonce,
     dangerouslySetInnerHTML: {
       __html: criticalCss
     }
diff --git a/node_modules/@remix-run/react/dist/esm/components.js b/node_modules/@remix-run/react/dist/esm/components.js
index e71a019..693f658 100644
--- a/node_modules/@remix-run/react/dist/esm/components.js
+++ b/node_modules/@remix-run/react/dist/esm/components.js
@@ -167,7 +167,7 @@ function composeEventHandlers(theirHandler, ourHandler) {
  *
  * @see https://remix.run/components/links
  */
-function Links() {
+function Links({ nonce }) {
   let {
     manifest,
     routeModules,
@@ -180,6 +180,7 @@ function Links() {
   let matches = errors ? routerMatches.slice(0, routerMatches.findIndex(m => errors[m.route.id]) + 1) : routerMatches;
   let keyedLinks = React.useMemo(() => getKeyedLinksForMatches(matches, routeModules, manifest), [matches, routeModules, manifest]);
   return /*#__PURE__*/React.createElement(React.Fragment, null, criticalCss ? /*#__PURE__*/React.createElement("style", {
+    nonce,
     dangerouslySetInnerHTML: {
       __html: criticalCss
     }
